src/core/io/src/4C_io_discretization_visualization_writer_mesh.cpp

```diff
@@ -166,11 +166,19 @@ namespace Core::IO
         }
         case OutputEntity::element:
         {
+          if (unique_names.size() != 1)
+            FOUR_C_THROW(
```

Comment:
Thanks for the suggestion! I personally still prefer the current way since for me it is easier to see right away when the check is triggered (but maybe it is just because I am getting old...).